API/Data/Offer.go

```diff
@@ -1,11 +1,22 @@
 package Data
 
 type OfferCreate struct {
-	Title        string `json:"title" form:"title"`
-	Description  string `json:"description" form:"description"`
-	Location     string `json:"location" form:"location"`
-	SalaryMin    uint32 `json:"salary_min" form:"salary_min"`
-	SalaryMax    uint32 `json:"salary_max" form:"salary_max"`
-	CompanyId    uint32 `json:"company_id" form:"company_id"`
-	ContractType uint32 `json:"contract_type" form:"contract_type"`
+	Title        string   `json:"title" form:"title"`
+	Description  string   `json:"description" form:"description"`
+	Location     string   `json:"location" form:"location"`
+	SalaryMin    uint32   `json:"salary_min" form:"salary_min"`
+	SalaryMax    uint32   `json:"salary_max" form:"salary_max"`
+	CompanyId    uint32   `json:"company_id" form:"company_id"`
+	ContractType []uint32 `json:"contract_type" form:"contract_type"`
+}
+
+type OfferUpdate struct {
+	Id           uint32   `json:"id" form:"id"`
+	Title        string   `json:"title" form:"title"`
+	Description  string   `json:"description" form:"description"`
+	Location     string   `json:"location" form:"location"`
+	SalaryMin    uint32   `json:"salary_min" form:"salary_min"`
+	SalaryMax    uint32   `json:"salary_max" form:"salary_max"`
+	CompanyId    uint32   `json:"company_id" form:"company_id"`
+	ContractType []uint32 `json:"contract_type" form:"contract_type"`
```

Comment:
Au lieu de faire une list d'ID, je pense que tu peux peut-être faire directement :
```suggestion
	ContractType []*Models.ContractType `json:"contract_type" form:"contract_type"`
```

Reply:
Le Json ne connait pas le type en db je pense, je testerai quand même.

Reply:
Ah oui pardon effectivement.